### ISSUE TITLE
Receptacle M-RCNN label set

### ISSUE THREAD

**Issue by Axe--** [soyeonm/FILM#4]
Hey, for the pretrained **receptacle Mask RCNN**, where is the corresponding label set (classes) that contains the **32** categories?

The `alfworld_receptacles` variable contains 38 classes, and upon visualizing the bboxes, the labels predicted are incorrect (mismatch).

Could you assist with the same, Thanks!

```
def load_pretrained_model(path, device, which_type):
    if which_type == 'obj':
        categories = len(object_detector_objs)
    elif which_type =='recep':
        categories = 32
    mask_rcnn = get_model_instance_segmentation(categories+1)
    #pickle.dump(torch.load(path, map_location=device), open("loaded.p", "wb"))
    mask_rcnn.load_state_dict(torch.load(path, map_location=device))
    return mask_rcnn
```

**Comment by soyeonm** [comment]
Hello, sorry for my late reply. I will reply back in a few days!

**Comment by joeyy5588** [comment]
I think you can find this one in https://github.com/alfworld/alfworld/blob/master/alfworld/gen/constants.py#L1206 since they are using the pretrained detector from this repo.

**Comment by ByZ0e** [comment]
Hi! Having the same confusion. It seems that only 16 receptacle objects are segmented. What are they and what are the correspondences between them and the visualization color palette?
Thanks :)
